eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -195,12 +201,22 @@ stages:
                           node ./eng/tools/versioning/increment.js --artifact-name ${{ artifact.name }} --repo-root .
                         displayName: Increment package version
 
+                      - bash: |
+                          node common/scripts/install-run-rush.js install
+                        displayName: "Install dependencies"
+
+                      - bash: |
+                          npm install -g ./common/tools/dev-tool
+                          npm install ./eng/tools/eng-package-utils
+                          node ./eng/tools/eng-package-utils/update-samples.js ${{ artifact.name }}
+                        displayName: Update samples
+
                       - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
                         parameters:
                           RepoName: azure-sdk-for-js
-                          PRBranchName: increment-package-version-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
-                          CommitMsg: "Increment package version after release of ${{ artifact.name }}"
-                          PRTitle: "Increment version for ${{ parameters.ServiceDirectory }} releases"
+                          PRBranchName: post-release-automation-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
+                          CommitMsg: "Post release automated changes for ${{ artifact.name }}"
+                          PRTitle: "Post release automated changes for ${{ parameters.ServiceDirectory }} releases"
```

Comment:
@praveenkuttappan Where is the PR description set? I was looking at https://github.com/Azure/azure-sdk-for-js/pull/16961 and was thinking that it would be good for the PR description to list changes that the automation is doing